cylc/flow/network/schema.py

```diff
@@ -1398,45 +1404,43 @@ class Flow(String):
             environment variables.
 
-            A broadcast can target any [runtime] namespace for all cycles or
-            for a specific cycle. If a task is affected by specific-cycle and
-            all-cycle broadcasts at once, the specific takes precedence. If
-            a task is affected by broadcasts to multiple ancestor
-            namespaces, the result is determined by normal [runtime]
-            inheritance. In other words, it follows this order:
-
-            `all:root -> all:FAM -> all:task -> tag:root -> tag:FAM ->
-            tag:task`
-
-            Broadcasts persist, even across restarts, until they expire
-            when their target cycle point is older than the oldest current in
-            the workflow, or until they are explicitly cancelled with this
-            command.  All-cycle broadcasts do not expire.
-
-            For each task the final effect of all broadcasts to all namespaces
-            is computed on the fly just prior to job submission.  The
-            `--cancel` and `--clear` options simply cancel (remove) active
-            broadcasts, they do not act directly on the final task-level
-            result. Consequently, for example, you cannot broadcast to "all
-            cycles except Tn" with an all-cycle broadcast followed by a cancel
-            to Tn (there is no direct broadcast to Tn to cancel); and you
-            cannot broadcast to "all members of FAMILY except member_n" with a
-            general broadcast to FAMILY followed by a cancel to member_n (there
-            is no direct broadcast to member_n to cancel).
+            A broadcast can set/override any `[runtime]` configuration for all
+            cycles or for a specific cycle. If a task is affected by a
+            specific-cycle and an all-cycle broadcasts at the same time, the
+            specific takes precedence.
+
+            Broadcasts can also target all tasks, specific tasks or families of
+            tasks. If a task is affected by broadcasts to multiple ancestor
+            namespaces (tasks it inherits from), the result is determined by
+            normal `[runtime]` inheritance. In other words, it follows this
+            order:
+
+            ```
+            all:root -> all:FAM -> all:task -> tag:root -> tag:FAM -> tag:task
+            ```
+
+            Broadcasts are applied at the time of job submission.
+
+            Broadcasts persist, even across restarts. Broadcasts made to
+            specific cycle points will expire when the cycle point is older
+            than the oldest active cycle point in the workflow.
+
+            Active broadcasts can be revoked using the "clear" mode.
+            Any broadcasts matching the specified cycle points and
+            namespaces will be revoked.
+
+            Note you can't broadcast to all cycles or namespaces *except* X
+            by broadcasting to all cycles and then clearing that broadcast for
+            cycle X.
```

Comment:
I still find this a bit confusing.
```suggestion
            namespaces will be revoked. This means you cannot remove
            individual cycles/namespaces from an all-cycle/namespace broadcast.
```

Reply:
The original wording suggests you can do that, by broadcasting to all, and then clearing one.

Maybe phrase it positively? Something like this: *to exclude specific cycles or namespaces from a general broadcast, target them all then explicitly clear the specific ones*

Reply:
> ... _to exclude specific cycles or namespaces from a general broadcast, target them all then explicitly clear the specific ones_

My reading was that you **cannot** do that

Reply:
> My reading was that you cannot do that

Correct, but since it caused confusion I've reworded more clearly.

Reply:
Oops, you're right. Actually I don't think it was confusing, but my brain evidently reordered the original text on the fly to get *you can't do blah X except by blah* rather than *you can't do "blah except X" by blah* (more or less).